.github/workflows/release.yml

```
@@ -170,11 +170,11 @@ jobs:
     #####################################################################
     - name: Get current date
       id: date
-      run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+      run: echo "date=$(date +'%Y-%m-%d')" >> $env:GITHUB_OUTPUT
       
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)